content/applications/general/developer_mode.rst

```diff
@@ -4,76 +4,80 @@
+Go to :menuselection:`Settings --> General Settings --> Developer Tools` and click on
+:guilabel:`Activate the developer mode`. At least one application needs to be installed for the
+:guilabel:`Developer Tools` section to appear in the :guilabel:`Settings` module.
 
 .. image:: developer_mode/settings.png
    :align: center
-   :alt: Overview of the debug options under settings in Odoo
+   :alt: Overview of the debug options under settings in Odoo.
 
 .. note::
-   *Activate the developer mode (with assets)* is used by developers; *Activate the developer mode
-   (with tests assets)* is used by developers and testers.
+   :guilabel:`Activate the developer mode (with assets)` is used by developers; :guilabel:`Activate
+   the developer mode (with tests assets)` is used by developers and testers.
 
-Once activated, the *Deactivate the developer mode* option becomes available.
+Once activated, the :guilabel:`Deactivate the developer mode` option becomes available.
 
 Activate through a browser extension
 ====================================
 
-| Go to the settings and extensions of your web browser, and search for *Odoo Debug*. Once the
-  extension is installed, a new icon will be shown on your toolbar.
-| For the *Odoo Debug* extension, a single click enables a normal version of the mode, while a
-  double click enables it with assets. To deactivate it, use a single click.
+Within a web browser, navigate to the browser settings and extensions and search for the `Odoo
+Debug` extension. Once the extension is installed, a new icon will be shown on the browser's
+toolbar.
 
-.. image:: developer_mode/monkey.png
+For the *Odoo Debug* extension, a single click enables a normal version of the mode, while a
+double click enables it with assets. To deactivate it, use a single click.
+
+.. image:: developer_mode/developer-mode-monkey.png
    :align: center
-   :alt: View of odoo’s debug icon in a chrome’s toolbar
+   :alt: View of Odoo's debug icon in a Google Chrome toolbar.
 
 Activate through the command palette
 ====================================
 
-The command palette tool has a command to activate the debug mode: open it with
-the keyboard shortcut `ctrl+k`, then type `debug`: a command will show up to
-activate the debug mode. 
+In Odoo the command palette tool has a command to activate the debug mode. First, open the command
+palette tool with the keyboard shortcut `ctrl+k` and then type `debug`. A command will show up to
+activate the debug mode.
 
-.. image:: developer_mode/command_palette.png
+.. image:: developer_mode/command-palette.png
    :align: center
-   :alt: Command palette with debug command
+   :alt: Command palette with debug command.
 
 Activate through the URL
 ========================
 
-In the URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the
-debug mode, add `?debug=0` instead.
+In the URL, add ``?debug=1`` or ``?debug=true`` after *web*. To deactivate the debug mode, add
```

Comment:
Add `The debug mode can also be activated by adding an extra query string to the database's URL.` before the sentence `In the URL...after *web*.`